docs/about-cedar/browser-support/README.md

```diff
@@ -0,0 +1,22 @@
+---
+{
+  "title": "Browser Support Matrix",
+  "title_metadata": false,
+  "layout": "LayoutArticle",
+  "summary": false,
+}
+---
+
+# Supported Platforms
+
+## Browsers
+
+Cedar supports the latest, stable releases of the following browsers and platforms. Support is not guaranteed for platforms beyond this list, and any issues reported against unsupported platforms will be investigated at the team’s earliest opportunity.
+Officially supported platforms: are:
+
+| **Platform** | **Chrome** | **Firefox** | **Safari**     |
+| :----------- | :--------- | :---------- | :------------- |
+| Windows 10   | Yes        | Yes         |                |
+| Windows 7    | Yes        |             |                |
+| Mobile iOS   |            |             | Mobile version |
+| OS X 10.13   | Yes        |             | Yes            |
```

Comment:
Are we really not supporting IE or Edge?

Reply:
thats a question for Test and Zack